app/atoms/base/_forms.scss

```diff
@@ -1,22 +0,0 @@
-input {
```

Comment:
Just create a new issue for this https://github.com/moxie-leean/ng-web-app/issues/59